### ISSUE TITLE
LaTeX: macro-only LaTeX blocks render as invisible

### ISSUE THREAD

**Issue by jimrandomh** [LessWrong2/Lesswrong2#749]
Vadim Kosoy writes in https://www.alignmentforum.org/posts/JiMAMNAb55Qq24nES/announcing-alignmentforum-org-beta#ArHSahkwjL6sbNurJ

> Not sure whether this is the right place to voice technical complaints, but. I am unhappy about the handling of LaTeX macros (on which I rely heavily). Currently it seems like you can add macros either as inline equations or as block equations, and these macros are indeed available in the following equations. However, if an equation object contains only macros, it is invisible and seems to be impossible to edit after creation. As a workaround, I can add some text + macros into the same equation object, but this is very hacky. It would be nice if either equation objects with macros would remain visible, or (probably better) there would be a special "header" in each post where I can put the macros. It would be even more amazing if you could load LaTeX packages in that header, but that's supererogatory.

Making a special LaTeX header object would complicate the UI and be tricky. Making empty-rendering LaTeX objects visible would be easier; ideally, we'd make them visible in the post editor but not in the rendered page. This may involve diving into draft.js, however.

**Comment by Discordius** [comment]
Yeah, not sure how difficult this would be. We haven't forked the
draft-js-mathjax package that we use, so we either have to do that, or
submit a PR to that package and hope that it goes through.
